lib/idx/flow/FlowMonitor.ts

```diff
@@ -31,6 +31,12 @@ export class FlowMonitor {
       return false;
     }
 
+    const previousValues = Object.keys(this.previousRemediator.values);
```

Comment:
I think this block belongs in a flow-specific FlowMonitor, like `RegistrationFlowMonitor`. 

```javascript
export class RegistrationFlowMonitor extends FlowMonitor {
  loopDetected (remediator: Remediator): boolean {
    // block here

    return super.loopDetected();
  }

Reply:
Authenticator enrollment can also happen in `AuthenticationFlow` - it looks like we might want to introduce a common superclass for these flows and move this block there.

Reply:
addressed in https://github.com/okta/okta-auth-js/pull/1026/commits/2aaeb238a5e1eb71bd0699ec342435b8d0537748